### ISSUE TITLE
So.. This is Rambox, but Open Source and without tracking?

### ISSUE THREAD

**Issue by salemaziel** [TheGoddessInari/hamsket#165]
If so, awesome.

Just discovered Rambox today, it looked kinda cool until I checked their privacy policy:

> We collect personal information that you provide to us such as name, address, contact information, passwords and security data, payment information, and social media login data....
...We may collect limited data from public databases, marketing partners, social media platforms, and other outside sources.... [Info may ] include: social media profile information (your name, gender, birthday, email, current city, state and country, user identification numbers for your contacts, profile picture URL and any other information that you choose to make public); marketing leads and search results and links, including paid listings

What in the  hell could they possibly need all this sh*t for? My fkn address? HA.

> We only share information with your consent, to comply with laws, to protect your rights, or to **fulfill business obligations**.

So.. sell every scrap of data about me, as well as their souls.
No thanks Rambox. Not today, Satan.

I will be checking out Hamsket this week. If you can confirm that Hamsket is indeed Rambox but without all the BS, I will donate forsure

**Comment by salemaziel** [comment]
Do you have paypal? the gofundme page doesn't seem to have that option

**Comment by flying-sheep** [comment]
> real life has been absolute hell lately.

Really sorry to hear that. If money helps, sure, go for the sponsors page. If not, please take care!

**Comment by jonaspm** [comment]
There is a "sponsor <3" button right at the top of the Github repo page.

**Comment by flying-sheep** [comment]
Yes, as of 14e0934411bba37dbda33cdd6d4ef30cf7d4f89a, which has been committed one month ago, i.e. more than two weeks after @TheGoddessInari told us about it.
